airflow/providers/amazon/aws/hooks/athena.py

```diff
@@ -91,8 +94,12 @@ def run_query(
         }
         if client_request_token:
             params["ClientRequestToken"] = client_request_token
+        if self.log_query:
+            self.log.info("Running Query with params: %s", params)
```

Comment:
Would it be a good idea to still log the query, but to the debug level instead?

Reply:
We only log query details when option `log_query = True`, like `DbApiHook` option `log_sql`.
So it may be better to keep using `info` level.